api/db-cdr/generate-cloudsql-cdr.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# This generates new cloudsql database for a cdr
+# note dev-up must be run to generate the schema
+# note run-local-data-migrations must be run to generate hard coded data from liquibase
```

Comment:
My recommendation is to merge this  / call this conditionally from run-local-data-migrations... that way we can be sure that this script gets invoked immediately after loading the data when the user wants that to happen.